content/doc/addons/cellar.md

```diff
@@ -17,279 +16,321 @@ aliases:
+  {{< /tab >}}
 
-Import the AWS SDK S3 library.
-With maven, it can be done with the following dependency :
+{{< tab >}}
+  **Python**
 
-```xml
-<dependency>
-  <groupId>software.amazon.awssdk</groupId>
-  <artifactId>s3</artifactId>
-  <version>2.21.35</version>
-</dependency>
-```
+  Tested with Python 3.6.
 
-Make sur to use latest version of the `2.X`, new versions are released regularly. See [the AWS Java SDK Documentation](https://github.com/aws/aws-sdk-java-v2/#using-the-sdk) for more details.
-
-Below is a sample Java class, written in Java 21, listing the objects of all buckets :
-
-```java
-import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
-import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
-import software.amazon.awssdk.services.s3.S3Client;
-import software.amazon.awssdk.services.s3.model.Bucket;
-import software.amazon.awssdk.services.s3.model.ListObjectsRequest;
-
-import java.net.URI;
-import java.util.List;
-
-public class CleverCloudCellarDemoApplication {
-
-    // replace those values with your own keys, load them from properties or env vars
-    private static final String CELLAR_HOST = "";
-    private static final String CELLAR_KEY_ID = "";
-    private static final String CELLAR_KEY_SECRET = "";
-
-    public static void main(String[] args) {
-        // initialize credentials with Cellar Key ID and Secret
-        // you can also use `EnvironmentVariableCredentialsProvider` by setting AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY env vars
-        var credentialsProvider = StaticCredentialsProvider.create(AwsBasicCredentials.create(CELLAR_KEY_ID, CELLAR_KEY_SECRET));
-
-        // create a client builder
-        var s3ClientBuilder = S3Client.builder()
-                // override the S3 endpoint with the cellar Host (starting with 'https://'
-                .endpointOverride(URI.create(CELLAR_HOST))
-                .credentialsProvider(credentialsProvider);
-
-        // initialize the s3 client
-        try (S3Client s3 = s3ClientBuilder.build()) {
-            // list buckets
-            List<Bucket> buckets = s3.listBuckets().buckets();
-            buckets.forEach(bucket -> {
-                // list bucket objects
-                var listObjectsRequest = ListObjectsRequest.builder().bucket(bucket.name()).build();
-                var objects = s3.listObjects(listObjectsRequest).contents();
-                // handle results
-            });
-
-        }
-    }
-}
-```
-
-See the [AWS Java SDK code examples for S3](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/s3) for more example use cases.
+  This script uses boto, the old implementation of the aws-sdk in python. Make sure to not use boto3, the API is completely different. The host endpoint is `cellar-c2.services.clever-cloud.com` (verify the `CELLAR_ADDON_HOST` variable value in the Clever Cloud console, from the **Information** option).
```

Comment:
It could be great to update this example or not tell customers not to use boto3?